templates/outputs.yml

```diff
@@ -3,4 +3,4 @@ Outputs:
   AutoScalingGroupName:
     Value: { Ref: AgentAutoScaleGroup }
   InstanceRoleName:
-    Value: { Ref: InstanceRoleName }
+    Value: { GetAtt: IAMRole.RoleName }
```

Comment:
The CloudFormation docs show a slightly different function invocation format:

```yaml
!GetRef AgentAutoScaleGroup
!GetAtt IAMRole.RoleName
```

source: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/outputs-section-structure.html

Reply:
Yup, should be either:

`{ "Fn::GetAtt" : [ "IAMRole", "RoleName" ] }`

Or:

`!GetAtt IAMRole.RoleName`